### PR TITLE
preview-registry: add my own key for pchickey namespace

### PR DESCRIPTION
**Pull request by pchickey**

### CODE REVIEW

**Review by calvinrp — APPROVED** [review]

File: authorized_keys.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [namespace_keys]
 "peterhuene" = ["sha256:1e4642089cefa79e792451b8d5e782ae6712e55ca8eea38cf56c92793faa2e90"]
+"pch" = ["sha256:6cc3bbe999b7fab65ad57bdb870e138e0780e5805cba42e816c0e38b4cbdb386"]
```
Comment (peterhuene) [inline authorized_keys.toml:3]: But `pch` is _also_ my initials, good sir.

I request trial by combat.

**Comment by pchickey** [comment]
fourth person involved and we collided in a 26^3 namespace!

**Comment by peterhuene** [comment]
That was (mostly - I wouldn't pass up an opportunity to battle with the Kirk vs. Spock theme song playing, though) a joke, but it's probably a good idea to have descriptive names so we know the owner of the keys.

To that end, perhaps it's a good idea to comment on each line with the owner's github handle, or something?

**Review by peterhuene — APPROVED** [review]

**Comment by pchickey** [comment]
Of course!

I guess I'd expect that, in the longer run, that we'd associate keys with usernames, and then associate usernames with groups which are allowed to publish in namespaces.

**Pull request merged**